app/styles/atoms/attribute.less

```diff
@@ -1,5 +1,6 @@
 .oss-attribute {
   height: 24px;
+  min-height: 24px;
```

Comment:
is this really needed ? 🤔

Reply:
Not sure depending on the use-case . I'd keep it simply because there is no impact if they are displayed in a column like in the designs but if we display them in a row , without this they start overflowing if the label or the value are too long. Just a small adjustment I added when checking the display on mobile.

Reply:
Then let's go for the trio ^^, can you add `max-height: 24px` ?